ui/src/components/Layout/styles.module.scss

```diff
@@ -0,0 +1,26 @@
+.container {
+  min-height: 100vh;
+  width: 100vw;
+  background-color: #f4f5f7;
+  display: flex;
+}
+.mainContent {
+  width: 82%;
+}
+.mainContentBlur::after {
+  content: '\A';
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  background: rgba(0, 0, 0, 0.6);
+}
+
+@import '../../index';
```

Comment:
put imports at the top. It's easier for the reader to know at one glance what all is imported instead of looking through the whole file.